complete page and wiki overhaul (#84)
betaflight/betaflight.com: CONTRIBUTION.md

# Contribution guide

## Development

For developing please read **[README.md](/README.md)**.

## Docs

> For writing docs you should the MDX reference on the docusaurus page [MDX Reference](https://docusaurus.io/docs/markdown-features/react).

Reviewer: ```suggestion
For writing docs please read the MDX reference on the docusaurus page [MDX Reference](https://docusaurus.io/docs/markdown-features/react).
```
